plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -429,5 +430,16 @@ public static String getDeploymentConfigNameForPods(List<IPod> pods) {
 				.map(pod -> pod.getLabels().get(DEPLOYMENT_CONFIG_KEY))
 				.orElse(null);
 	}
+	
+	public static com.openshift.restclient.model.IProject getProject(IResource resource) {
```

Comment:
There are 2 IProject classes: one from Eclipse and the other from Openshift REST client